numpy/lib/tests/test_format.py

```diff
@@ -838,7 +838,11 @@ def test_large_file_support():
 
 @dec.slow
 def test_large_archive():
-    a = np.empty((2 ** 30, 2), dtype=np.uint8)
+    try:
+        a = np.empty((2**26, 2), dtype=np.uint8)
```

Comment:
Hmm, is this big enough? The test is for
```
    BUG: Cast size to int64 when loading from archive
    
    Prevents overflow errors for large arrays on systems
    where the default int type is int32.
```
IIRC, the fix was for the product of dimensions that overflowed, so the tested array should probably be multidimensional with total size >= 2**31